feat: Modal migration guides (#1499)
contentful/forma-36: MIGRATION.md

  <GridItem />
</Grid>;
```

## Modal

> API of Modal didn't change a lot in the version 4. The way of using Modal still stayed the same. The main logical change is moving buttons from the left to right side in the `<Modal.Controls>` component. That's why we recommend to swap action buttons - primary action should the one that is displayed as a first one from the right in the Modal. Have a look how to do it:

Reviewer: ```suggestion
In version 4 the Modal component buttons were moved from the left to right side of the `<Modal.Controls>` component. We now recommend swapping action buttons, the primary action should be displayed as the first from the right in the Modal. Have a look how to do it:
```

> In the version 3:

Reviewer: ```suggestion
In version 3:
```

```tsx
<Modal.Controls>
  <Button variant="primary" />
  <Button variant="secondary" />
</Modal.Controls>
```

> In the version 4:

Reviewer: ```suggestion
In version 4:
```

```tsx
<Modal.Controls>
  <Button variant="secondary" size="small" />
  <Button variant="primary" size="small" />
</Modal.Controls>
```

### How to migrate your Modal components

Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `Modal` component to the new version:

`npx @contentful/f36-codemod`

> If you decide to do it manually, you would need to transform you existing code:

Reviewer: ```suggestion
If you want to do it manually, you must transform your existing code as follows:
```
